programmers/난이도별/level01.나누어_떨어지는_숫자_배열/sjjyy.java

```diff
@@ -0,0 +1,36 @@
+package 난이도별.level01.나누어_떨어지는_숫자_배열;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class sjjyy
+{
+    public static int[] solution(int[] arr, int divisor) {
+        List<Integer> List = new ArrayList<>();
+
+        for (int value : arr) {
+            if (value % divisor == 0)
+                List.add(value);
+        }
+
+        if (List.isEmpty())
+            List.add(-1);
+
+        int[] ans = new int[List.size()];
+
+        for (int i = 0 ; i < List.size() ; i++)
+            ans[i]= List.get(i);
+
+        Arrays.sort(ans);
+
+        return ans;
```

Comment:
```suggestion
        int[] answer = Arrays.stream(arr)
                             .boxed()
                             .filter(v -> v % divisor == 0)
                             .sorted((a, b) -> a - b)
                             .mapToInt(Integer::valueOf)
                             .toArray();
        return answer.length == 0 ? new int[]{-1} : answer;
```

Stream을 조금 더 유연하게 사용하면 이렇게 풀이할 수 있습니다~

Arrays는 import 해줘야돼요!